### PR TITLE
adding note about requiring 3-character usernames

### PR DESCRIPTION
**Pull request by mikegreen**
Related to
https://app.asana.com/0/1143232772343007/1146683609014113
This adds language to configure Okta with a scheme that will ensure >=3 character usernames

### CODE REVIEW

File: content/source/docs/enterprise/saml/identity-provider-configuration-okta.html.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Follow these steps to configure Okta as the identity provider (IdP) for Terrafor
     - **Name ID format** (drop-down): EmailAddress (the full name for this format in the SAML specification is `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`).
     - **Application username** (drop-down): Email
 
+    ~> **Important:** TFE usernames must be at least 3 characters. If Email is not used as the application username, ensure that your Okta username or other method selected contains only usernames having at least 3 characters. For example, if Email Prefix is selected and the email is Jo@domain.com, the user will not be able to login and receive the message "ERROR: Validation failed: Username must have at least 3 characters"
```
Comment (nfagerlund) [inline content/source/docs/enterprise/saml/identity-provider-configuration-okta.html.md]:
```suggestion
    ~> **Important:** TFE usernames must be at least 3 characters. If Email is not used as the application username, ensure that your Okta username or other method selected contains only usernames having at least 3 characters. For example, if Email Prefix is selected and the email is Jo@domain.com, the user will not be able to log in and will receive the message "ERROR: Validation failed: Username must have at least 3 characters".
```

**Comment by nfagerlund** [comment]
I'm gonna assume that minor tweak was OK and just ship this!

**Comment by dylanegan** [comment]
@nfagerlund I believe this username restriction was removed and anything with a length of at least one character will work.

**Comment by nfagerlund** [comment]
@dylanegan 😭

**Pull request merged**